test/bootstrap_test.go

```diff
@@ -59,18 +57,9 @@ func TestBootstrapError(t *testing.T) {
 
 	switch err := errors.Cause(err).(type) {
 	case *testGateway.MalformedStdoutError:
-		var lineStruct struct {
-			Msg   string
-			Error string
-		}
-		jsonErr := json.Unmarshal([]byte(err.StdoutLine), &lineStruct)
```

Comment:
This expected log output to be a valid json. ie,
`{"msg":...., "err:..."}.

However, tchannel produces an extra logger.info line and the log output changes to
'{"msg":..., "err:..."}
{"msg":..., "err:..."}`
which is not valid json. I changed the test to just check for substring because that is sufficient for this purpose